tool/resources/org/antlr/v4/tool/templates/codegen/Go/Go.stg

```diff
@@ -702,8 +700,7 @@ ListLabelName(label) ::="<label>"
 CaptureNextToken(d) ::= "<d.varName> = p.GetTokenStream().LT(1)"
 CaptureNextTokenType(d) ::= "<d.varName> = p.GetTokenStream().LA(1);"
 
-StructDecl(struct, ctorAttrs, attrs, getters, dispatchMethods, interfaces, extensionMembers, tokenDecls, tokenTypeDecls,
-	tokenListDecls, ruleContextDecls, ruleContextListDecls, attributeDecls, superClass={ParserRuleContext}) ::= <<
+StructDecl(struct, ctorAttrs, attrs, getters, dispatchMethods, interfaces, extensionMembers, tokenDecls, tokenTypeDecls, tokenListDecls, ruleContextDecls, ruleContextListDecls, attributeDecls, superClass={ParserRuleContext}) ::= <<
```

Comment:
231 characters, I'd prefer 2 or 3 lines (120 columns limit). Or even 14 lines, but it seems I am the only one in this world who likes one argument each line when the argument list is very long. 

This is just bike-shedding. LGTM. :)